packages/base-styles/_variables.scss

```diff
@@ -57,7 +57,7 @@ $admin-bar-height-big: 46px;
 $admin-sidebar-width: 160px;
 $admin-sidebar-width-big: 190px;
 $admin-sidebar-width-collapsed: 36px;
-$modal-min-width: 360px;
+$modal-min-width: 350px;
```

Comment:
Needs rebase to remove this from the changeset.

Reply:
Rebased https://github.com/WordPress/gutenberg/pull/50978/commits/ba7681b718840128eb329579a5556c7c83ffdc8b